### PR TITLE
my code review suggestions

### PR DESCRIPTION
**Pull request by Odogwufu**
Description of Project: App for socializing with other people. With the ability for users to create posts and also share links with other users and change the text font, color, and background for any created posts.

Was the program available in UC Github on time?
Yes, the program was available.

does the program build?
Yes, the program was built successfully

rational behind changes:
One change I made was fixing a typo in the HTML Startpage. And then I also added a bootstrap button. Since they had linked the bootstrap style sheet. It wasn't difficult to add. The next change I made was to add a style.css file to the templates directory. This will allow them to easily change the background on the home page and later on change the format of the forms they used on the HTML page. After looking at their java classes and controllers. They followed every convention correctly so there wasn't much I could add.

links to three commits I made:
484515b39d8e46054714205088ba15a457ccefae
63bc9b8d9df6c1edaa8fd39a89a2e00f7552f5fe
b3ef0b5f840f61dcb96cb3ad649db3949675bb79

technical concepts I learned
1. I learned how to use bootstrap buttons in HTML
2. I learned more about Autowired and component 
3. I learned how to implement classes.

### CODE REVIEW

File: src/main/resources/templates/startPage.html [hunk 1]
```diff
@@ -16,7 +17,7 @@ <h1>Simple Talk</h1>
     <br />
     <form action="#"  method="GET">
         <label for="searchTerm">Search</label><input type="text" name="searchTerm" id="searchTerm"/>
-        <button name="submit" tyoe="submit" value="0">Search</button>
+        <button class="btn btn-secondary" button name="submit" type="submit" value="0">Search</button>
```
Comment (Odogwufu) [inline src/main/resources/templates/startPage.html:20]: added bootstrap buttons.https://getbootstrap.com/docs/4.0/components/buttons/

File: src/main/resources/templates/style.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+body {
```
Comment (Odogwufu) [inline src/main/resources/templates/style.css:1]: added css file.https://www.w3schools.com/html/html5_syntax.asp

File: src/main/resources/templates/startPage.html [hunk 3]
```diff
@@ -16,7 +17,7 @@ <h1>Simple Talk</h1>
     <br />
     <form action="#"  method="GET">
         <label for="searchTerm">Search</label><input type="text" name="searchTerm" id="searchTerm"/>
-        <button name="submit" tyoe="submit" value="0">Search</button>
```
Comment (Odogwufu) [inline src/main/resources/templates/startPage.html:19]: fixed naming convention. https://www.w3schools.com/html/html5_syntax.asp

File: src/main/resources/templates/style.css (hunk 2 above)
Comment (discospiff) [reply src/main/resources/templates/style.css:1]: Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

**Comment by discospiff** [comment]
These changes are mostly UI related, which isn't necessarily technical debt reduction for a Spring Boot project.  I recommend that the group review the changes and decide what to keep.

**Pull request closed without merging**